src/main/java/frc/robot/subsystems/arm/Gripper.java

```diff
@@ -43,7 +43,7 @@ public class Gripper extends AbstractArmPart {
 
   private DigitalInput m_coralSensorBack = new DigitalInput(IoPortsConstants.CoralChannelIDBack);
 
-  private Debouncer m_coralSensorDebouncerFront = new Debouncer(GripperConstants.CoralDropDebounceSeconds, DebounceType.kBoth);
+  private Debouncer m_coralSensorDebouncerFront = new Debouncer(0.1, DebounceType.kBoth); // TODO: what do we do here?
```

Comment:
We probably want different debounce values for front and back (and might need to change our scoring logic to account for it)